modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Functions to find and manipulate content.
+ *
+ * @example
+ * var contentLib = require('/lib/xp/content-graphql');
+ *
+ * @module content
+ */
+
+/* global __ */
+
+function required(params, name) {
+    var value = params[name];
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)